app/javascript/components/new-product-form.vue

```diff
@@ -137,6 +137,25 @@
         >
           {{ product ? 'GUARDAR' : 'AGREGAR' }}
         </button>
+        <form
+          method="post"
+          :action="`/stores/catalog/${product.id}`"
+          @submit="confirmDelete"
+        >
+          <input type="hidden" name="_method" value="delete" />
```

Comment:
Disallow self-closing on HTML void elements (<input/>)  vue/html-self-closing<br>Attribute "name" should be on a new line                vue/max-attributes-per-line<br>Attribute "value" should be on a new line               vue/max-attributes-per-line